pkg/controller/config/config_controller.go

```diff
@@ -233,6 +233,22 @@ func (r *ReconcileConfig) reconcileInstall(req reconcile.Request, res *op.Config
 			Version: tektonVersion})
 		return reconcile.Result{}, err
 	}
+
+	for _, path := range res.Spec.AddOns {
+		extension, err := mf.NewManifest(filepath.Join("deploy", "resources", path), true, r.client)
+		if err != nil {
```

Comment:
wouldn't you want to update the status of installation, just like we have done for `pipeline` see `updateStatus`